bundles/org.eclipse.orion.client.ui/web/orion/parameterCollectors.js

```diff
@@ -328,6 +318,24 @@ define(['i18n!orion/nls/messages', 'orion/webui/littlelib'],
 					localClose();
 					if (typeof(cancelFunction) === 'function') cancelFunction();
 				}, false);
+
+				if (commandInvocation.parameters.hasOptionalParameters()) {
+					commandInvocation.parameters.optionsRequested = false;
+
+					var options = makeButton(messages["More"], parentDismiss); //$NON-NLS-0$
+					var activateMoreOptions = function() {
+						commandInvocation.parameters.optionsRequested = true;
+						finish(self);
+					}
+					if (Object.keys(commandInvocation.parameters.parameterTable).length === 0) {
+						activateMoreOptions();
+					} else {
+						options.addEventListener("click", function() { //$NON-NLS-0$
```

Comment:
activeMoreOptions is already a function, so you should be able to just pass it in, right?